server/src/routes/api/v1/ReviewsRouter.js

```diff
@@ -24,4 +24,14 @@ reviewsRouter.post('/', async (req, res) => {
   }
 })
 
+reviewsRouter.delete("/:id", async (req,res)=>{
+  try {
+    await Review.query().findById(req.params.id).delete()
```

Comment:
ideally, on the back end here we also check that the current user `req.user` is the same as the user who created the review, and if so go ahead with deleting the review. where possible we want to have consistent validations on the front end and back end